sipnet.c

```diff
@@ -882,10 +882,24 @@ void freeClimateList(int numLocs) {
 // rather than returning a value, they have as parameters the variable(s) which they modify
 // so a single function can modify multiple variables
 
-
-// calculate amount of light absorbed
-// using light attenuation (as in PnET)
-// CURRENTLY UNUSED (see calcLightEff3)
+/**
+ * @brief Compute light absorbed using simple integration
+ * 
+ * **CURRENTLY UNUSED** (see calcLightEff3)
+ * canopy light effect (simple integration).
+ *
+ * This uses light attenuation (as in PnET) to first calculate light intensity and then the light effect `lightEFF`
+ * for each layer.
+ * 
+ * cumLai is 0 at the canopy top and equals total LAI at the bottom.
+ * 
+ * Note: SIPNET is not a multi-layer model; this function derives a canopy‐averaged 
+ * light effect that is then used to calculate calculate GPP for the whole canopy.
```

Comment:
calculate is repeated
```suggestion
 * light effect that is then used to calculate GPP for the whole canopy.
```